src/__tests__/integration.test.ts

```diff
@@ -858,3 +858,85 @@ describe('.Integrations', () => {
     `)
   })
 })
+
+describe('queryString', () => {
```

Comment:
This file is getting **very** large.
How about you create a `query-string.integration.test.ts`?